CONTRIBUTING.md

```diff
@@ -30,6 +30,8 @@ Please select the `master` branch as the destination for your Pull Request so yo
 
 Please squash your changes into a single commit using a command like `git rebase -i upstream/master`.
 
+To test C++ changes, you can compile the module using `npm run compile`.
```

Comment:
Oh, you’re right, I didn’t know `npm install` runs `node-gyp` on the package itself. I always tried `npm rebuild` but that only recompiles dependencies. However, you said it ‘will […] test the changes’. This doesn’t happen for me (see below). Am I missing something?

```
npm i

> sharp@0.10.0 install /Users/dani/workspace/sharp
> node-gyp rebuild

  CXX(target) Release/obj.target/sharp/src/common.o
  CC(target) Release/obj.target/sharp/src/composite.o
  CXX(target) Release/obj.target/sharp/src/utilities.o
  CXX(target) Release/obj.target/sharp/src/metadata.o
  CXX(target) Release/obj.target/sharp/src/resize.o
  CXX(target) Release/obj.target/sharp/src/sharp.o
  SOLINK_MODULE(target) Release/sharp.node
  SOLINK_MODULE(target) Release/sharp.node: Finished
```